app/react-components/Projects.js

```diff
@@ -20,12 +20,12 @@ const Projects = ({
   genesIsFetching,
   numUniqueCases,
 }: {
-    projects: Array<Object>,
-    topGenesWithCasesPerProject: { [gene_id: string]: { [project_id: string]: number, symbol: string}},
     numUniqueCases: number,
     FacetService: any,
```

Comment:
if FacetService doesn't have a flow type you can import yet can type this from these lines? 

```
            (acc2, s) => acc2 + FacetService.getActiveIDs('primary_site').indexOf(s), 0) !== 0) {
            p.primary_site.map(s => FacetService.addTerm('primary_site', s));
          } else {
            p.primary_site.map(s => FacetService.removeTerm('primary_site', s));
          }
        },
        outer: [
          ...(acc[p.primary_site] || { outer: [] }).outer,
          {
            key: p.project_id,
            value: p.summary.case_count,
            tooltip: `<b>${p.name}</b><br />${p.summary.case_count} case${p.summary.case_count > 1 ? 's' : 0}`,
            clickHandler: () => {
              if (FacetService.getActiveIDs('project_id').indexOf(p.project_id) !== 0) {
                FacetService.addTerm('project_id', p.project_id);
              } else {
                FacetService.removeTerm('project_id', p.project_id);
```